src/github/mod.rs

```diff
@@ -400,6 +617,60 @@ impl Diff {
     }
 }
 
+enum RepoDiff {
+    Create(CreateRepoDiff),
+    Update(UpdateRepoDiff),
+}
```

Comment:
Any reason why this is not like this?

```rust
enum RepoDiff {
  CreateRepo(..),
  UpdateDescription(..),
  CreatePermission(..),
  UpdatePermission(..),
  DeletePermission(..),
}
```

Reply:
This makes the diffs order dependent which I would like to avoid.

Reply:
Aren't diffs order dependent already? If the same team repo commit creates a team and adds it to a GitHub repo, if you run the repo diff before the team diff applying the repo diff will fail.